relayer/cmd/root.go

```diff
@@ -24,6 +24,12 @@ var rootCmd = &cobra.Command{
 func init() {
 	cobra.OnInitialize(initConfig)
 	rootCmd.PersistentFlags().StringVar(&configFile, "config", "", "config file")
+	rootCmd.PersistentFlags().Int(
+		"direction",
+		0,
+		"Relay messages bi-directionally (0), from Eth to Sub (1), or from Sub to Eth (2)",
+	)
+	rootCmd.PersistentFlags().Bool("headers", false, "Only forward headers")
```

Comment:
I'd rename this flag to `--headers-only`.

As an aside, in a future iteration we will probably want to convert this into a `mode` flag which allows one to specify headers, events, or both.

Reply:
Something to consider: most events probably cannot be validated without the node being able to cross-reference the relevant header/block. So, if you run the relay with events only, you'll need to sync headers some other way

Reply:
Yeh all the events stuff will be deprecated/modified to support verification with cross reference one we have transaction receipt proofs working so np for now